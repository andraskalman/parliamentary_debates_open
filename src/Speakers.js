import React from 'react';

const Speakers = ({ speakers }) => (
  <ul>
    {speakers.map(speaker => (
      <li key={speaker}>{speaker}</li>
    ))}
  </ul>
)

export default Speakers
