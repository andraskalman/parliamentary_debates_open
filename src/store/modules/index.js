import { combineReducers } from 'redux'
import { routerReducer } from 'react-router-redux'
import search from './search'
import help from './help'

export default combineReducers({
  routing: routerReducer,
  search,
  help
})
