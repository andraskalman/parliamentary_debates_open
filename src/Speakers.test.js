import React from 'react';
import ReactDOM from 'react-dom';
import Speakers from './Speakers';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux'
import store, { history } from './store'
import { ConnectedRouter } from 'react-router-redux'


it('renders correctly', () => {
  const tree = renderer
    .create(<Speakers
        speakers={{"John Doe": true, "Random Name": false}}
        counts={{"John Doe": 1, "Random Name": 2}} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
})
