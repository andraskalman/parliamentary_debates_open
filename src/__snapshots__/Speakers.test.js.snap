// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ul>
  <li>
    John Doe
  </li>
  <li>
    Random Name
  </li>
</ul>
`;
